classes/openrc-image.bbclass

```diff
@@ -50,6 +50,34 @@ openrc_add_services() {
     done
 }
 
+openrc_disable_services() {
+    for pair in ${OPENRC_DISABLED_SERVICES}; do
+        runlevel=${pair%%:*}
+        svc=${pair##*:}
+
+        svcfile="${IMAGE_ROOTFS}${sysconfdir}/runlevels/${runlevel}/${svc}"
+        if ! [ -L "$svcfile" ]; then
+            bbfatal "No openrc service named '${svc}' found in runlevel '${runlevel}."
+        fi
+        rm "$svcfile"
+    done
+}
+
+openrc_delete_services() {
```

Comment:
The main use-case I had for this one in particular is to remove some default scripts that are part of the OpenRC packages itself, which aren't installed using `openrc.bbclass` and associated variables and functions. It also provides a way to customize it on an image level instead of pure recipe append (e.g., when we're building an initramfs it should have almost no services, but in our main images we do want some) -- hope that makes sense.

Reply:
You'd be right that in the second case a lot would be mitigated by not installing the package to begin with in the initramfs, and that when you need the package disabling has the same effect, some minor space costs aside.

If you think the first case is better served by a .bbappend and `do_install:append()` in the `openrc` package, that is fine by me too, I mostly added this to not have to worry about the specific logic of where services are located when I want to delete services.

Reply:
Got it, the initramfs case makes sense.  I was trying to figure out when you'd want to remove services that openrc itself had an opinion on.  In that case my only request is that you put this rationale in the commit message so that I can remember this in a year.  Well, that and add a default value with documentation, the same as `OPENRC_DISABLED_SERVICES`.

Reply:
Thanks, will do!